controller/scheduler-v2/scheduler.go

```diff
@@ -297,6 +364,7 @@ func (s *Scheduler) stopJob(req *JobRequest) (err error) {
 		}
 		s.sendEvent(NewEvent(EventTypeJobStop, err, nil))
 	}()
+	//FIXME: HostID and JobID are most likely empty right now
```

Comment:
If `JobID` is not set, this should find all the formation jobs of the given type and deterministically pick one to stop (e.g. the one created most recently)